tests/test_main.py

```diff
@@ -29,3 +57,15 @@ def test_main(_mock1, _mock2, _mock3, _mock4):
     """Test the function main."""
     resp = main()
     assert (isinstance(resp, dict))
+
+
+@mock.patch('f8a_report.main.ReportHelper', return_value=MockReportHelper)
+@mock.patch('f8a_report.main.manifest_interface', return_value=True)
+@freeze_time("2020-04-06")
+def test_environment(_mock1, _mock2):
```

Comment:
You have to write a unit test for negative case, i.e. for `GENERATE_MANIFESTS=false`.

Reply:
os.environ.get('GENERATE_MANIFESTS') can be true or false.. can't be equality checks or distinct negative checks. 
However, it will fail if there is anything other than two expected values